docs/ja/faq.md

```
@@ -5,125 +5,131 @@ outline: deep
----
-
-## データ量とパフォーマンス
+* [データ量とパフォーマンス](#データ量とパフォーマンス)
+  
+* [FIWARE Orion との違い](#fiware-orion-との違い)
+  
+* [デプロイと運用](#デプロイと運用)
+  
+* [API の使い方](#api-の使い方)
+  
+* [地理空間拡張](#geospatial-extensions)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)